discord/ext/bridge/context.py

```diff
@@ -50,14 +50,15 @@ class BridgeContext(ABC):
 
         @bot.bridge_command()
         async def example(ctx: BridgeContext):
-            if isinstance(ctx, BridgeExtContext):
-                command_type = "Traditional (prefix-based) command"
-            elif isinstance(ctx, BridgeApplicationContext):
+            if ctx.is_app:
                 command_type = "Application command"
+            else:
+                command_type = "Traditional (prefix-based) command"
             await ctx.send(f"This command was invoked with a(n) {command_type}.")
 
     .. versionadded:: 2.0
     """
+    is_app: bool
```

Comment:
Alternative idea for this: make `is_app` a property, and return the value of `isinstance(self, BridgeApplicationContext)`